Xamarin.Forms.Platform.Android/ViewRenderer.cs

```diff
@@ -92,10 +92,16 @@ protected override void Dispose(bool disposing)
 					_container = null;
 				}
 
-				if (Element != null && _focusChangeHandler != null)
+				if (Element != null)
 				{
-					Element.FocusChangeRequested -= _focusChangeHandler;
-					_focusChangeHandler = null;
+					Element.Behaviors?.Clear();
+					Element.Triggers?.Clear();
```

Comment:
the XF team doesn't make any assumption about Element reuse. Elements can be reused, and are reused. But an element can not be used multiple times simultaneously, i.e. have more than one Parent.